app/src/main/java/com/asfoundation/wallet/billing/adyen/AdyenPaymentViewModel.kt

```diff
@@ -99,6 +102,8 @@ class AdyenPaymentViewModel @Inject constructor(
   val singleEventState = _singleEventState.receiveAsFlow()
   lateinit var paymentStateEnum: PaymentStateEnum
   var cancelPaypalLaunch = false
+  var runningCustomTab = false
```

Comment:
this is always false

Reply:
This var is changed inside Fragment When lauch CustomTabs